### DISCUSSION TITLE
开源项目孵化、托管、运营机制

### DISCUSSION THREAD

**Discussion by TechQuery** [kaiyuanshe/RFC#2] [Ideas]
## 现有项目

https://kaiyuanshe.cn/categories/Project/

![image](https://user-images.githubusercontent.com/19969570/102222869-f4966d80-3f1e-11eb-8c71-2a67bb021a02.png)

## 机制提纲

- 捐赠：开源协议、法务 @WallaceOS
- 导师：社区建设 @tedliu1 @legist
- 基建：云服务基础设施 @zhuangbiaowei @juniwang
- 营销：媒体组宣传推广 @cynbarby

**Comment by tedliu1** [comment]
水歌，闪电侠⚡太牛了

**Comment by JPyer** [comment]
I also want to join the "open source society". But why is there a problem with the entrance of the official website? What are the other ways to join?

**Reply by TechQuery** [reply]
Thank for your report. But you may post an Issue in one of the following repositories:
- Official Web site: https://github.com/kaiyuanshe/kaiyuanshe.github.io
- Member system: https://github.com/kaiyuanshe/web-members

then, this comment will be removed.

**Reply by TechQuery** [reply]
http://members.kaiyuanshe.cn/ is working now, try again please.

**Comment by TechQuery** [comment]
- 提案文档：https://kaiyuanshe.feishu.cn/wiki/NY1BwrZR8iCaBWk3HE2cHoYyn4m
- 开源代码：https://github.com/idea2app/GitHub-reward
